packages/mobile/package.json

```diff
@@ -9,11 +9,10 @@
     "start:bg": "react-native start &",
     "start:android": "react-native run-android",
     "start:ios": "react-native run-ios",
-    "lint-checks": "yarn run lint && yarn run build:typescript",
     "lint": "tslint -c tslint.json --project tsconfig.json",
-    "build": "yarn run build:typescript && yarn run build:metro",
-    "build:sdk": "build-sdk",
-    "build:typescript": "tsc --noEmit",
+    "build": "yarn run build:ts && yarn run build:metro",
+    "build:sdk": "yarn --cwd ../contractkit build:env",
```

Comment:
is `env` in `build:env` a variable here?

Reply:
No, just the name of the npm script. You would still call `yarn build:sdk alfajores`.
I would prefer not to have a n npm script in mobile that call contractkit, but i saw that it was being use in a deploy script

Reply:
Ah okay. In that case, should it be build:for-env to be consistent with verification pool?

Reply:
yeah. i like it.